runner/local/simple.go

```diff
@@ -2,69 +2,136 @@ package local
 
 import (
 	"fmt"
+	"sync"
+	"time"
+
 	"github.com/scootdev/scoot/runner"
 	"github.com/scootdev/scoot/runner/execer"
-	"sync"
 )
 
 func NewSimpleRunner(exec execer.Execer) runner.Runner {
+	return NewMultiRunner(exec, 1)
+}
+
+func NewMultiRunner(exec execer.Execer, maxRunning int) runner.Runner {
```

Comment:
Why the change to support running multiple simultaneously?

This seems like a big change that makes things more complicated (e.g., we can no longer just have one checkout we reuse, we have to allocate). Is there something it enables right now?

I didn't see tests for this more complex functionality, but I think they're important. Did I miss them, or have you not done them yet?